changelog_unreleased/javascript/10417.md

```diff
@@ -1,4 +1,4 @@
-#### Support Module Blocks proposal (#10417 by @sosukesuzuki)
+#### Support Module Blocks proposal (#10417 by @sosukesuzuki & thorn0)
```

Comment:
nits (I cannot use code suggestion because I created this PR)
```diff
- #### Support Module Blocks proposal (#10417 by @sosukesuzuki & thorn0)
+ #### Support Module Blocks proposal (#10417 by @sosukesuzuki, @thorn0)
```